pkg/cli/ui_test.go

```diff
@@ -106,16 +105,16 @@ func TestUI_Confirm(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.test, func(t *testing.T) {
 			ctx := createTestContext()
-			ctx.io.InBuf.WriteString(c.input)
+			ctx.io.In.(*bytes.Buffer).WriteString(c.input)
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Using the variable on range scope `c` in function literal (scopelint)